src/mocks/msw/handlers.ts

```diff
@@ -0,0 +1,71 @@
+import {
+  BackendDestinationEnum,
+  getEndpoint,
+} from '../../lib/utils/functions/getEndpoint'
+import { mockSchemaBinding } from '../mockSchema'
+import { rest } from 'msw'
+import {
+  mockFileEntityBundle,
+  mockProjectEntityBundle,
+  MOCK_FILE_ENTITY_ID,
+  MOCK_PROJECT_ID,
+} from '../entity/mockEntity'
+import { mockUserBundle, mockUserProfileData, MOCK_USER_ID } from '../user/mock_user_profile'
+
+const handlers = [
+  rest.options('*', async (req, res, ctx) => {
+    return res(ctx.status(200))
+  }),
+
+  rest.get(
+    `${getEndpoint(
+      BackendDestinationEnum.REPO_ENDPOINT,
+    )}repo/v1/userProfile/:id`,
```

Comment:
In the near future, I'd like to 
* Reference API paths using constants, so these handlers reference the same values as SynapseClient
* Split these handlers up and group them by related services, just to keep things tidy